docs-ref-conceptual/includes/cli-install-linux-dnf.md

```diff
@@ -9,10 +9,10 @@ ms.custom: devx-track-azurecli
 
 ## Overview
 
-For Linux distributions with `dnf` such as RHEL 8/9 or CentOS Stream 8/9, there are RPM packages for Azure CLI.
+For Linux distributions with `dnf`, such as RHEL 8/9 or CentOS Stream 8/9, there are RPM packages for Azure CLI.
 
 > [!WARNING]
-> Starting from Azure CLI 2.39.0 (released on 08/02/2022), no RPM packages will be released for RHEL 7 and CentOS 7. You may continue to use historical versions of Azure CLI on these systems, but there will be no further updates or bug fixes from Microsoft for those versions.
+> Azure CLI 2.39.0 (released on 08/02/2022) is the last version supported on RHEL 7 and Centos 7 and will continue to receive security fixes until the end of life of the operating systems. Microsoft will not provide any other updates or bug fixes on this version.
```

Comment:
This changes the meaning of the original statement. The original statement indicates 2.39.0 doesn't support RHEL 7 and CentOS 7, but the new statement indicates the opposite.